bats/tests/preferences/move-from-roaming-to-local.bats

```diff
@@ -0,0 +1,42 @@
+load '../helpers/load'
+
+local_setup() {
+    skip_on_unix 'roaming appdata => local appdata migration is windows-only'
+    ROAMING_HOME="$(win32env APPDATA)/rancher-desktop"
+}
+
+@test 'factory reset' {
+    factory_reset
+}
+
+@test 'start app, create a setting, and move settings to roaming' {
+    start_container_engine
+    wait_for_container_engine
+
+    rdctl api -X PUT /settings --body '{ "version": '"$(get_setting .version)"', "WSL": {"integrations": { "beaker" : true }}}'
```

Comment:
I wonder if one day we will validate the WSL distro names, and then throw an error on this request, but we can worry about that when we actually do.